tests/route/test_forced_mgmt_route.py

```diff
@@ -33,27 +33,45 @@ def backup_restore_config(duthosts, enum_rand_one_per_hwsku_hostname):
     restore_config(duthost, CONFIG_DB, CONFIG_DB_BACKUP)
 
 
-def get_interface_reload_timestamp(duthost):
-    timestamp = duthost.command("sudo systemctl show --no-pager interfaces-config"
-                                " -p ExecMainExitTimestamp --value")["stdout"]
-    logger.info("interfaces config timestamp {}".format(timestamp))
+def get_file_hash(duthost, file):
```

Comment:
>get_file_hash [](http://example.com/codeflow?start=4&length=13)

Better to combine sha1 and file timestamp together to prevent any hash confliction risk.

Reply:
Fixed